### PR TITLE
Release/0.1.0 alpha.17

### PR DESCRIPTION
**Pull request by colmsnowplow**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,13 +1,18 @@
+Version 0.1.0-alpha.167 (2020-05-04)
```
Comment (paulboocock) [inline CHANGELOG]: Typo, should be alpha.17

**Review by paulboocock — APPROVED** [review]
LGTM. I’ll let @AlexBenny comment on the FMDB version updates with iOS 1.3.0 - I think the updates here are correct but I recall some issue in 1.3.0 with FMDB so worth a second pair of eyes.

File: CHANGELOG (hunk 1 above)
Comment (colmsnowplow) [reply CHANGELOG]: Ah! Thanks.

**Review by AlexBenny — APPROVED** [review]
I haven't tested it on my machine but it looks good to me! 🚢 
About the issue with FMDB, it's mostly related to Swift Package Manager so it's not an issue on ReactNative, however it will be released soon a patch version 1.3.1 of the iOS tracker.

**Pull request merged**